package/rmservewacominput/package

```diff
@@ -23,6 +31,7 @@ build() {
 
 package() {
     install -D -m 755 "$srcdir"/c_implementation/rmServeWacomInput "$pkgdir"/opt/bin/rmservewacominput
-    install -D -m 644 "$srcdir"/oxide "$pkgdir"/opt/etc/draft/rmservewacominput
+    install -D -m 755 "$srcdir"/rmservewacominput_gui "$pkgdir"/opt/bin/rmservewacominput_gui
```

Comment:
Underscores are pretty uncommon in executable names. I’d suggest using `rmservewacominput-gui` instead. (Needs to be changed in the draft unit file too.)

```suggestion
    install -D -m 755 "$srcdir"/rmservewacominput_gui "$pkgdir"/opt/bin/rmservewacominput-gui
```